docs/examples/objc/iOS-Carthage/ThemisTest/AppDelegate.m

```diff
@@ -27,155 +27,212 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // We don't do UI. Please look into debug console to see the results.
     //
 
+    // Please, look in debug console to see results
+
+    // Generating/reading keys:
+    
     [self runExampleGeneratingKeys];
-    [self runExampleReadingKeysFromFile];
+    [self readingKeysFromFile];
 
+    
+    // Secure Message:
+    
     [self runExampleSecureMessageEncryptionDecryption];
     [self runExampleSecureMessageSignVerify];
-
-    [self runExampleSecureCellSealMode];
-    [self runExampleSecureCellTokenProtectMode];
-    [self runExampleSecureCellImprint];
-
+    
+    
+    // Secure Cell:
+    
+    NSData * fixedKey = [[NSData alloc]
+                         initWithBase64EncodedString:@"UkVDMgAAAC13PCVZAKOczZXUpvkhsC+xvwWnv3CLmlG0Wzy8ZBMnT+2yx/dg"
+                         options:NSDataBase64DecodingIgnoreUnknownCharacters];
+    [self runExampleSecureCellSealMode:fixedKey];
+    [self runExampleSecureCellTokenProtectMode:fixedKey];
+    [self runExampleSecureCellImprint:fixedKey];
+    
+    
+    NSData * generatedKey = TSGenerateSymmetricKey();
+    [self runExampleSecureCellSealMode:generatedKey];
+    [self runExampleSecureCellTokenProtectMode:generatedKey];
+    [self runExampleSecureCellImprint:generatedKey];
+    
+    
+    // Secure Comparator
     [self runExampleSecureComparator];
 
     return YES;
 }
 
 #pragma mark - Secure Cell
 
-- (NSData *)generateMasterKey {
-    NSString *masterKeyString = @"UkVDMgAAAC13PCVZAKOczZXUpvkhsC+xvwWnv3CLmlG0Wzy8ZBMnT+2yx/dg";
-    NSData *masterKeyData = [[NSData alloc] initWithBase64EncodedString:masterKeyString
-                                                                options:NSDataBase64DecodingIgnoreUnknownCharacters];
-    return masterKeyData;
-}
-
-- (void)runExampleSecureCellSealMode {
+- (void)runExampleSecureCellSealMode:(NSData *)masterKeyData {
     NSLog(@"----------------- %s -----------------", sel_getName(_cmd));
-
-    NSData *masterKeyData = [self generateMasterKey];
-    TSCellSeal *cellSeal = [[TSCellSeal alloc] initWithKey:masterKeyData];
+    
+    TSCellSeal * cellSeal = [[TSCellSeal alloc] initWithKey:masterKeyData];
 
     if (!cellSeal) {
         NSLog(@"%s Error occurred while initializing object cellSeal", sel_getName(_cmd));
         return;
     }
 
-    NSString *message = @"All your base are belong to us!";
-    NSString *context = @"For great justice";
-    NSError *themisError;
+    NSString * message = @"All your base are belong to us!";
+    NSString * context = @"For great justice";
+    NSError * themisError;
```

Comment:
Ｉｓ　ｔｈｉｓ　ｒｅａｌｌｙ　ｔｈｅ　ｐｒｅｆｅｒｒｅｄ　ｃｏｄｅ　ｓｔｙｌｅ　？

I mean , Apple docs definitely don ' t use punctuation like this . It also brings a lot of diff noise .

Reply:
¯\_(ツ)_/¯ 
i aligned the code styles with that one that we've used in other examples.

Reply:
Okay, my fault for not adding code formatters. `clang-format` should do the trick for Objective-C and `swift-format` should work for Swift.

Well, I can agree to tolerate this but only because a) there is no formal code style, b) there are no formatters, c) a merit of unifying the style is displayed. Though, I can challenge the last point:

| Project | Code style |
| -- | -- |
| `docs/examples/objc/iOS-Carthage` | :one:  `NSString *message` |
| `docs/examples/objc/iOS-CocoaPods` | :two: `NSString * message` |
| `docs/examples/objc/macOS-Carthage` | :one: `NSString *message` |
| `docs/examples/Themis-server/Obj-C` | :two: `NSString * base64Body` |
| `src/wrappers/themis/Obj-C/objcthemis` | :one: `NSMutableData *unwrappedMessage` |
| `tests/objcthemis/objthemis` | :two: `NSString * sharedSecret` |

(So it's actually a tie. I won't count the lines.)

But FWIW, I'd personally keep whatever format is there in the files and use the same format for new code added there. The only acceptable reason for massive diff noise is some automated reformatting to the format which is going to be enforced later. If that's the thing, it should be in a separate PR anyway, not in the one which adds some new examples.